lib/seahorn/BvOpSem2MemRepr.cc

```diff
@@ -52,6 +52,9 @@ Expr OpSemMemArrayRepr::MemCpy(Expr dPtr, Expr sPtr, unsigned len,
       Expr val = op::array::select(srcMem, sIdx);
       res = op::array::store(res, dIdx, val);
     }
+  } else {
+    ERR << "Word size and pointer are not aligned!";
```

Comment:
`ERR` usually entails an assertion following it

Reply:
Ack. Added assert.